bstrp_functions.sh

```diff
@@ -26,26 +26,14 @@ get_cmake_version(){
   CMAKE_REVISION=$(echo "$CMAKE_VERSION" | cut -d. -f3)
 }
 
-add_option(){
-  eval "$1=$2"
-  OPTIONS+=("$1")
-  CMAKE_OPTIONS_ENABLED+=("$1:$3")
-  CMAKE_OPTIONS_DISABLED+=("$1:$4")
-}
-
-add_enabled_option(){
-  eval "$1=$2"
-  OPTIONS+=("$1")
-  CMAKE_OPTIONS_DISABLED+=("$1:$3")
-}
 add_cmake_option(){
   eval "$1=$2"
 }
 
-add_disabled_option(){
+add_option(){
```

Comment:
Would you mind naming the parameters, like this:
```suggestion
add_option(){
  BOOTSTRAP_OPTION_NAME="$1"
  DEFAULT_VALUE="$2"
  CMAKE_OPTION_NAME="$3"
```
and using these names thoughout the rest of the function?

Reply:
Good idea, added in 828ae6d371e4ec88eb29b2cafbf701a7d8d89009